ci/playbooks/deploy_podified_openstack.yaml

```diff
@@ -55,3 +59,10 @@
         oc get secrets;
         oc rsh openstackclient openstack compute service list;
         oc rsh openstackclient openstack network agent list;
+<<<<<<< HEAD
```

Comment:
you have an unresolved merge conflict here

Reply:
Done